inst/rmarkdown/templates/jds/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,271 @@
+  <!-- mathematical variables should be in italic, vectors in boldface. Units, -->
+  <!-- abbreviations and special functions should be upright. Please add notes to -->
+  <!-- explain any other special symbols. -->
+<!-- \item Figures: Figures should be original laser prints with high contrast, -->
+  <!-- suitable for immediate reproduction, typed on separate sheets and identified -->
+  <!-- by its number. They will normally be reduced to one column width (6--8 cm). In -->
+  <!-- the figures, the main lines should be about 0.3 mm in width, and the assistant -->
+  <!-- lines 0.15 mm. Notations in the figures should be distinct and consistent with -->
+  <!-- the same ones in the text, and their font size will be 7--9 pt. The positions -->
+  <!-- of figures should be marked in the text by boxes of a suitable size. Each -->
+  <!-- figure should have its own caption. For photographs, the original photos must -->
+  <!-- be supplied with good contrast and clearly distinguishable details. -->
+<!-- \item Tables: Tables, numbered in order of appearance, should be appended on -->
+  <!-- separate sheets and identified with appropriate titles.The table title, which -->
+  <!-- should be brief, goes above the table. A detailed description of its contents -->
+  <!-- or table footnotes should be given directly below the body of the table. -->
+<!-- \item References: References must be published work, and numbered consecutively -->
+  <!-- in order of their first citation. References should be listed individually at -->
+  <!-- the end of the text and indicated in the text with a superscript number in -->
+  <!-- square brackets. All of the references’ authors, as well as the titles of the -->
+  <!-- referenced articles, should be given. -->
+<!-- \item Proofs: Authors will receive a letter informing them whether the -->
+  <!-- manuscript is accepted or rejected in two months. Authors should return their -->
+  <!-- revisions to the Manuscript Office within one month on receipt. When an -->
+  <!-- article has been amended in compliance with the comments of referee(s), an -->
+  <!-- electronic file of the final version should be sent with the revised -->
+  <!-- manuscript. Proofs will be sent to the authors and should be returned -->
+  <!-- preferably within 48 hours to avoid publication delays. -->
+<!-- \end{itemize} -->
+
+
+# A Placeholder Section {#sec:placeholder}
+
+\lipsum[4-7]
+
+# Citing References {#sec:citations}
+
+The citations are in the author-year format with the
+\texttt{chicago} bibstyle.
+
+A bibfile contains all the citations in
+bibtex format is prepared (see \texttt{JDSbib.bib}). Some characters
+in the title of the references needs to be protected so that the style
+file does not alter it. For example, in the bibtex entry for
+@Pozd:etal:2017, the ``B'' in ``Brownian'' and the ``E'' in
+``estimation''  following the colon are protected.
```

Comment:
FYI, the `smart` extension in Pandoc should be activated, and should take care of inserting those specific quotes

So this should also be possible to write with usual Markdown
```suggestion
@Pozd:etal:2017, the "B" in "Brownian" and the "E" in
"estimation" following the colon are protected.
```